Does not support trailing slash in server's URL

### Expected behavior

Works with `-p https://sonar.corpo.rate:9000/`

### Actual behavior

Fails with:
```
[ERROR] com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 12 path $
[ERROR] Server answered: <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<head>
  <meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
  <title>SonarQube (404)</title>
  <style type="text/css">
    body {
      background-color: #fff;
      color: #666;
      text-align: center;
      font-family: arial, sans-serif;
    }

    div.dialog {
      width: 25em;
      padding: 0 4em;
      margin: 4em auto 0 auto;
      border: 1px solid #ccc;
      border-right-color: #999;
      border-bottom-color: #999;
    }

    h1 {
      font-size: 100%;
      color: #f00;
      line-height: 1.5em;
    }
  </style>
</head>

<body>
<!-- This file lives in public/404.html -->
<div class="dialog">
  <h1>The page you were looking for doesn't exist.</h1>

  <p>You may have mistyped the address or the page may have moved.</p>

  <p>
    <a href="/">Go back to the homepage</a>
  </p>
</div>
</body>
</html>
 com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 12 path $
SonarQube online: false
[ERROR] Impossible to reach SonarQube instance.
```

### Steps to reproduce behavior

See above.

### Detection version
2.0

--- a/src/main/java/fr/cnes/sonar/report/model/SonarQubeServer.java
+++ b/src/main/java/fr/cnes/sonar/report/model/SonarQubeServer.java
@@ -63,6 +63,8 @@ public class SonarQubeServer {
      */
     public void setUrl(final String url) {
         this.url = url;
+        if (this.url.endsWith("/"))
+            this.url = this.url.substring(0, this.url.length() - 1);
     }
 
     /**

--- a/src/main/java/fr/cnes/sonar/report/model/SonarQubeServer.java
+++ b/src/main/java/fr/cnes/sonar/report/model/SonarQubeServer.java
@@ -63,8 +63,9 @@ public class SonarQubeServer {
      */
     public void setUrl(final String url) {
         this.url = url;
-        if (this.url.endsWith("/"))
+        if(this.url.endsWith("/")) {
             this.url = this.url.substring(0, this.url.length() - 1);
+        }
     }
 
     /**

